package.json

```diff
@@ -1,5 +1,4 @@
 {
-  "private": true,
```

Comment:
This isn't a private package, and CI publish script throws an error with the new yarn if this is still set to true. To match the previous behavior where CI does publish this package correctly to our internal artifactory, I had to remove this.